projects/cms-ui-demo/src/modules/validation-summarizer-demo/vms-with-basic-validator/vms-with-basic-validator.component.html

```diff
@@ -2,55 +2,249 @@
 <div class="row"
      [formGroup]="studentForm">
   <div class="col-12">
-    <div class="form-group">
-      <div class="card">
-        <div class="card-body">
-          <h5 class="card-title">Basic validation summarizer</h5>
-          <div class="card-text">
-            <ul>
-              <li>Validation summarizer can be placed under a specific control</li>
-              <li>It retrieves control errors and display them on the screen.</li>
-            </ul>
-          </div>
-          <div class="jumbotron">
-            <div class="row">
-              <div class="col-lg-12">
-
-                <!--Name-->
-                <label>{{'VALIDATION_SUMMARIZER_DEMO.NAME'}}</label>
-                <div class="form-group">
-                  <input class="form-control"
-                         name="studentName"
-                         [formControl]="studentNameControl">
-
-                  <cms-validation-summarizer [instance]="studentNameControl"
-                                             label="VALIDATION_SUMMARIZER_DEMO.NAME">
-                  </cms-validation-summarizer>
-                </div>
+    <!-- Title-->
+<!--    <h2 class="mt-5">-->
+<!--     {{'VALIDATION_SUMMARIZER_DEMO.BASIC_VALIDATOR' | translate}}-->
+<!--    </h2>-->
+    <!-- End title-->
+
+    <!-- Content-->
+    <div class="row my-4">
+      <!-- Left content-->
+      <div class="col-lg-3 col-md-4">
+        <div class="list-group">
+          <a (click)="scrollToItem('description')"
+             [class]="currentItem === 'description' ? listGroupActiveClass : listGroupInActiveClass">
+            {{'VALIDATION_SUMMARIZER_DEMO.BASIC_VALIDATOR_HOW_IT_WORK' | translate}}
+          </a>
+          <a (click)="scrollToItem('examples')"
```

Comment:
Do not hard code `examples` like this.
Move the definition into a class. For example:

```
export class ValidationSummarizerSections {
  public static readonly example = 'example';
}
```